apps/web/composables/useQuickCheckout/__tests__/useQuickCheckout.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuickCheckout } from '../useQuickCheckout';
+import { ProductMock } from '../../../__tests__/__mocks__/product.mock';
+
+describe('useQuickCheckout', () => {
+  it('should check if open is true', () => {
+    const { openQuickCheckout, quantity, isOpen, product } = useQuickCheckout();
+    openQuickCheckout(ProductMock, 3);
+    expect(isOpen.value).toBe(true);
+    expect(product.value).toStrictEqual(ProductMock);
```

Comment:
create own it for checking that the product is equal. always better to have only one expect in one it